docs/inventory.md

```diff
@@ -3,8 +3,8 @@
 The inventory file that the poller uses describes a set of sources used to gather the list of devices, credentials to authenticate in the devices, default settings and eventually puts all together defining namespaces. An extensive explanation of each secation is provided on this page.
 
 !!! warning
-    Starting with version 0.16.0 the old options `-D` and `-a`  and the old inventory format are no longer supported.
-    See the section [Migrating to the new format](#migrating-to-new-format).
+Starting with version 0.16.0 the old options `-D` and `-a` and the old inventory format are no longer supported.
+See the section [Migrating to the new format](#migrating-to-new-format).
```

Comment:
I think you have some kind of linter in your env. Can you disable it before updating the docs?

Reply:
Fixed.